api_app/analyzers_manager/admin.py

```diff
@@ -0,0 +1,3 @@
+# from django.contrib import admin
+
+# Register your models here.
```

Comment:
I guess it is already in your to-do list, but we would need admin interfaces against all the new models here.